swagger-templates/ruby/README.mustache

```diff
@@ -67,22 +67,17 @@ require '{{{gemName}}}'
 {{/authMethods}}end
 {{/hasAuthMethods}}
 
-api_instance = {{{moduleName}}}::{{{classname}}}.new{{#hasParams}}
-{{#vendorExtensions.x-codegen-hasRequiredParams}}{{#allParams}}{{#required}}
-{{{paramName}}} = {{{example}}} # {{{dataType}}} | {{{description}}}
-{{/required}}{{/allParams}}{{/vendorExtensions.x-codegen-hasRequiredParams}}{{#hasOptionalParams}}
-opts = { {{#allParams}}{{^required}}
-  {{{paramName}}}: {{{example}}}{{#vendorExtensions.x-codegen-hasMoreOptional}},{{/vendorExtensions.x-codegen-hasMoreOptional}} # {{{dataType}}} | {{{description}}}{{/required}}{{/allParams}}
-}{{/hasOptionalParams}}{{/hasParams}}
 
```

Comment:
I would hardcode the authorization as well because if the first endpoint doesn't require authentication, it will skip that part of the code.